### DISCUSSION TITLE
Jellyfin on Synology DS423 via docker

### DISCUSSION THREAD

**Discussion by cispadmin** [jellyfin/jellyfin#11295] [Q&A]
hi all!
Dear developers, I have a question that I could not resolve on my own.
I deployed jellyfin to synology DS 423 via docker.
While I was poking at different buttons, I took away my access rights to the server. To fix this, I deleted the container. reinstalled. nothing changed. I deleted the image and container - nothing changed.
Tell me how to find it, use shh to find and delete the directory monitored on the host.
The forum https://forum.jellyfin.org/t-the-video-is-slow-the-show-stops could not help me.
Please respond and tell me where to dig.
I wanted to post here all the code that I posted on the forum.
Why does everything look terrible?
Please follow the link, everything is more readable there.
Thanks anyway!

**Comment by cispadmin** [comment]
I may be wrong in my judgment, but in my opinion this is a bug.
If a person deletes the container, then there is no need to save the data. If he needs it, he will set everything up again.
Before putting this in a bug track, I would like to understand whether I am thinking correctly.
Well, besides, if this was intended, then it’s worth dropping at least a few words about it - like, my dear users, if you want to delete the config, then go here and delete

**Comment by gnattu** [comment]
![image](https://github.com/jellyfin/jellyfin/assets/12995396/ad8b634f-9022-4331-a639-bcb30f6554e9)

You REALLY need to ask on Synology forum for their specific docker implementation and not here.
